examples/fcis/train_coco_multi.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import division
+
+import argparse
+import numpy as np
+
+import chainer
+from chainer.training import extensions
+from chainer.training.triggers import ManualScheduleTrigger
+import chainermn
+
+from chainercv.chainer_experimental.datasets.sliceable \
+    import ConcatenatedDataset
+from chainercv.chainer_experimental.datasets.sliceable \
+    import TransformDataset
+from chainercv.chainer_experimental.training.extensions import make_shift
+from chainercv.datasets import coco_instance_segmentation_label_names
+from chainercv.datasets import COCOInstanceSegmentationDataset
+from chainercv.experimental.links import FCISResNet101
+from chainercv.experimental.links import FCISTrainChain
+from chainercv.experimental.links.model.fcis.utils.proposal_target_creator \
+    import ProposalTargetCreator
+from chainercv.extensions import InstanceSegmentationCOCOEvaluator
+from chainercv.links.model.ssd import GradientScaling
+
+from train_sbd import concat_examples
+from train_sbd import Transform
+
+
+def filter_dataset(dataset):
+    indices = []
```

Comment:
How about using the public interface of slicableDataset?
`indices = np.array([i for i, label in enumerate(dataset.slice[:, ['label']]) if len(label) > 0], dtype=np.int32)`